src/video-interface.js

```diff
@@ -105,10 +105,40 @@ export class VideoInterface {
    *
    * @param {string} unusedMethod
    * @param {function(!./service/action-impl.ActionInvocation)} unusedHandler
-   * @param {ActionTrust} minTrust
+   * @param {ActionTrust} unusedMinTrust
    * @public
    */
-  registerAction(unusedMethod, unusedHandler, minTrust) {}
+  registerAction(unusedMethod, unusedHandler, unusedMinTrust) {}
+}
+
+
+/**
+ * @interface
+ */
+export class VideoInterfaceWithAnalytics extends VideoInterface {
```

Comment:
There is a TODO to get rid of `getPlayedRanges` and infer it using the existing events and methods. At that point this simply becomes `getDuration()` and `getCurrentTime()` which is reasonable to expect everyone to implement, we can remove this interface then.